gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapTypeAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package com.google.android.gnd.ui.home.mapcontainer
+
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.gnd.R
+
+
+class MapTypeAdapter(
```

Comment:
Can you add kdoc for this class?

Reply:
Also please add minimal doc for all non-trivial public methods (e.g., "Returns the 'name' value calculated based on X Y and Z", not "Returns the 'name' value").

Reply:
Done